dianna/__init__.py

```diff
@@ -34,31 +33,30 @@
 
 
 def explain_image(model_or_function, input_data, method, labels, **kwargs):
-    """
-    Explain an image (input_data) given a model and a chosen method.
+    """Explain an image (input_data) given a model and a chosen method.
 
     Args:
         model_or_function (callable or str): The function that runs the model to be explained _or_
                                              the path to a ONNX model on disk.
         input_data (np.ndarray): Image data to be explained
         method (string): One of the supported methods: RISE, LIME or KernelSHAP
         labels (Iterable(int)): Labels to be explained
+        kwargs: These keyword parameters are passed on
 
     Returns:
         One heatmap (2D array) per class.
 
     """
     if method.upper() == "KERNELSHAP":
         # To avoid Access Violation on Windows with SHAP:
-        from onnx_tf.backend import prepare  # pylint: disable=import-outside-toplevel,unused-import
+        from onnx_tf.backend import prepare  # noqa: F401
```

Comment:
I would rather keep the human readable version. Is that something you can configure?
I'm surprised the linter instruction isn't removed at all for this line, because for instance in kernelshap and other places you are removing the ignore-unused-import instruction completely.

Reply:
Without the pragma, `ruff` replaces this entire import by `pass`, because the import is not used. This is what happened in all the other instances of `unused-import`.

For this instance I added back the noqa, because of the comment above. I guess there is some side effect from the import that dianna requires. 

I can't configure the noqa in a different way, this is the convention used by `ruff`.